libs/sdk-ui-ext/src/internal/components/dialogs/embedInsightDialog/EmbedInsightDialog.tsx

```diff
@@ -73,42 +76,47 @@ export const EmbedInsightDialog: React.VFC<IEmbedInsightDialogProps> = (props) =
             backend,
             colorPalette,
             executionConfig,
-            codeLang,
+            workspaceId,
         };
-
-        return codeOption.type === "definition"
-            ? generateCodeByDefinition({
+        if (!insight.insight.identifier) {
```

Comment:
Spread insight.insight.identifier and use in array dependency just identifier

Reply:
I think we don't need to spread it because the `insight` is still used in the `inputBase` in the same function. Thanks!